### PR TITLE
[OV] Fix dynamic shape for depthwise conv

### PR DESCRIPTION
**Pull request by kshpv**
### Changes

Add a check of dynamic shape to determine depthwise convolution.
Add pattern for OV

**Before:**

image

**After:**

image

### Reason for changes

Fix bug for rvm_mobilenet_v3

### Related tickets

107759

### Tests

TBD

### CODE REVIEW

**Comment by kshpv** [comment]
Conformance build is 99

File: nncf/experimental/openvino_native/hardware/fused_patterns.py [hunk 1]
```diff
@@ -399,6 +399,24 @@ def create_hswish_without_denominator():
...
 
 
+@OPENVINO_HW_FUSED_PATTERNS.register(PatternNames.SWISH_WITH_HARD_SIGMOID)
+def create_swish_with_hardsigmoid():
+    pattern = GraphPattern()
+    linear_node = pattern.add_node(**LINEAR_OPERATIONS)
+    add_node = pattern.add_node(**{GraphPattern.LABEL_ATTR: 'ADD_BIAS',
+                                   GraphPattern.METATYPE_ATTR: om.OVAddMetatype})
+    hard_sigmoid_node = pattern.add_node(**{GraphPattern.LABEL_ATTR: 'HARDSIGMOID',
+                                     GraphPattern.METATYPE_ATTR: om.OVHardSigmoidMetatype})
```
Comment (l-bat) [inline nncf/experimental/openvino_native/hardware/fused_patterns.py:409]:
```suggestion
    hard_sigmoid_node = pattern.add_node(**{GraphPattern.LABEL_ATTR: 'HARDSIGMOID',
                                            GraphPattern.METATYPE_ATTR: om.OVHardSigmoidMetatype})
```

**Review by l-bat — APPROVED** [review]

**Comment by kshpv** [comment]
run onnx pre-commit tests

**Comment by kshpv** [comment]
run pytorch pre-commit tests

File: nncf/experimental/openvino_native/hardware/fused_patterns.py [hunk 2]
```diff
@@ -399,6 +399,24 @@ def create_hswish_without_denominator():
     return pattern
 
 
+@OPENVINO_HW_FUSED_PATTERNS.register(PatternNames.SWISH_WITH_HARD_SIGMOID)
+def create_swish_with_hardsigmoid():
```
Comment (alexsu52) [inline nncf/experimental/openvino_native/hardware/fused_patterns.py:403]: Why did you add `swish_with_hardsigmoid` pattern only for OpenVINO backend?

Comment (MaximProshin) [reply nncf/experimental/openvino_native/hardware/fused_patterns.py:403]: please merge it as it is and open a new ticket for other BEs

Comment (alexsu52) [reply nncf/experimental/openvino_native/hardware/fused_patterns.py:403]: ticket: 109256
